rllib/__init__.py

```diff
@@ -28,8 +28,8 @@ def _setup_logger():
 
 
 def _register_all():
-    from ray.rllib.agents.trainer import Trainer
-    from ray.rllib.agents.registry import ALGORITHMS, get_trainer_class
```

Comment:
Lost short-term backward compatibility.

Reply:
There is a new test for this now that uses all the "old way" import statements based on older registry code.